proposals/0759-react-native-frameworks.md

```diff
@@ -0,0 +1,452 @@
+    - On the other hand, using Framework specific APIs requires library consumers to add an extra dependency in their application on the specific Framework. This could result in libraries that are less compatible with apps that are using other Frameworks.
+- Otherwise if you don’t want to impose a Framework dependency on their user, to ensure that your library is compatible with all Frameworks:
+  - The recommendation is to use the React Native Core API.
+
+We do acknowledge that the current recommendation for library authors is complicated, and the React Native team will be working with Frameworks developers in the future to simplify the current offerings.
+
+## Becoming a React Native Framework
+
+In this section, we clarify what are the **requirements** to become an official React Native Framework. We call this also the **Framework Contract** as it describes what are the benefits of- and the asks for being an official Framework.
+
+### Benefits
+
+Frameworks that are adhering to requirements described below:
+
+- Will be spotlighted in the Getting Started page of the React Native website (our highest traffic page). Specifically, we’ll recommend to users to use one of the Frameworks which satisfies the requirements.
+- Will be mentioned during public comms as one of the recommended frameworks by the React Native core team.
+- The company/entity behind the Framework will automatically be granted a [partner status](https://github.com/facebook/react-native/blob/main/ECOSYSTEM.md#partners).
+- PRs and Issues coming from the Framework developers will receive higher priorities.
+
+### Requirements
+
+Here is a short summary of what are the requirements for React Native Frameworks.
+
+#### DOs of Frameworks
+
+- **Focus on developer & user experience** - We believe Frameworks should aim to provide a top-notch developer experience. Frameworks that fail to provide good documentation or developer workflows are not likely high quality frameworks, and our users will most likely not want to use them. Similarly, we want to promote frameworks that provide a top-notch user experience and do not ignore basic user needs, such as accessibility, performance, and delightful UIs.
+- **Focus on building cohesive experiences** - Users are looking into deploying universal apps across multiple targets. We’re delegating this responsibility to frameworks that can provide better experiences to users and craft cohesive developer experiences between mobile, web, desktop and beyond.
+- **Focus on abstracting complexity away** - Any initiative that simplifies the developer workflow is welcomed in the framework domain.
+- **Be popular** - We’ll only be recommending solutions that are widely known and adopted by the community.
```

Comment:
How do we quantify that? (stars, npm downloads, or companies using it in production) Who's "we" in this context? I don't think this is something we have to answer right now, but I believe this can become more important as new frameworks are under way.